Detectors/ITSMFT/ITS/workflow/src/ThresholdCalibratorSpec.cxx

```diff
@@ -240,7 +251,7 @@ bool ITSCalibrator::findThreshold(
 // x is the array of charge injected values;
 // NPoints is the length of both arrays.
 // thresh, noise, chi2 pointers are updated with results from the fit
-bool ITSCalibrator::findThresholdFit(
+bool ITSThresholdCalibrator::findThresholdFit(
```

Comment:
Note that this method is not thread-safe due to the minuit used in the fit.  To avoid a race condition which is very difficult to debug, I would suggest to put somewhere in the initialization check
```
if (mFitType==FIT && mNThreads>1) {
  throw std::runtime_error("Multuple threads are requested with fit method");
}
```

Reply:
done